settings/base.py

```diff
@@ -4,7 +4,14 @@
 
 # Django settings file for a project based on the playdoh template.
 
-from funfactory.settings_base import *
+from funfactory.settings_base import (
+    path,
+    BASE_PASSWORD_HASHERS,
+    ROOT,
+    ROOT_URLCONF,
+    TEST_RUNNER
+)
```

Comment:
I think we need the "make elmo a proper python package", https://bugzil.la/934547 before going further down that route.